src/Api/Models/Api/Request/Accounts/PremiumRequestModel.cs

```diff
@@ -1,16 +1,15 @@
 ﻿using Microsoft.AspNetCore.Http;
 using System.ComponentModel.DataAnnotations;
 using System.Collections.Generic;
-using Bit.Core.Enums;
+using Enums = Bit.Core.Enums;
```

Comment:
Was this aliasing necessary?

Reply:
Yes, we have a property called `PaymentMethodType`. Which overrides the enum.